### PR TITLE
add igshid

### PR DESCRIPTION
**Pull request by jnozsc**
this tracking parameter is used as 
https://twitter.com/instagram/status/1217121431059869698

Screen Shot 2020-01-25 at 09 40 13

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -102,6 +103,17 @@ browser.webRequest.onBeforeRequest.addListener(
     ["blocking"]
 );
 
+// Filter out Instagram Share Identifier query parameters
+var clean_igshid = build_query_param_remover(f_match_igshid);
+browser.webRequest.onBeforeRequest.addListener(
+    clean_igshid,
+    {
+        urls: ["<all_urls>"],
```
Comment (Riamse) [inline background.js]: Is it necessarily the case that igshid appears on all urls, like fbclid does? Or just IG-internal urls?

Comment (jnozsc) [reply background.js]: make sense. fixed in https://github.com/apiraino/link_cleaner/pull/13/commits/b2a8e376b0731efedf6281a9bde296a2a57264c4

File: background.js [hunk 2]
```diff
@@ -92,6 +92,7 @@ browser.webRequest.onBeforeRequest.addListener(
     ["blocking"]
 );
 
+// Filter out Facebook Click Identifier query parameters
```
Comment (apiraino) [inline background.js]: thank you :-)

**Comment by apiraino** [comment]
hello @jnozsc thanks for the PR! 

Looks good to me, merging :+1:

File: background.js (hunk 1 above)
Comment (apiraino) [reply background.js]: :+1:

**Pull request merged**